### ISSUE TITLE
Alerts and other glyphicons and styling not rendered propertly when a PDF is generated

### ISSUE THREAD

**Issue by MrBenMyster** [dotnet/docfx#3016]
**DocFX Version Used**: 2.36.2

**Template used**: `pdf.default`

**Steps to Reproduce**:

1. Specify a markdown document which uses an "alert" like:
```
> [!NOTE]
> The word
```
2. Using default docfx.json settings for generating a pdf, build the docfx, which outputs an intermediate html file and a PDF file.

**Expected Behavior**:

The intermediate html file generated for PDF should render the Note like it does in the website, e.g.:
![image](https://user-images.githubusercontent.com/16561584/42402287-0422b126-8137-11e8-9148-6ec2e052b7a5.png)

**Actual Behavior**:

The PDF rendered the note like:
![image](https://user-images.githubusercontent.com/16561584/42402223-a47fd942-8136-11e8-92ab-bf03dc1c24e1.png)

**Notes**
I was able to get this scenario functional by customizing a template and loading the docfx.css into the pdf template.

**Comment by herohua** [comment]
@partychen, can you advise on how to customize the CSS for PDF template? Thanks!

**Comment by eBerdnA** [comment]
@MrBenMyster Could you please let me know which part you have customized? Thanks.

**Comment by MrBenMyster** [comment]
My apologies for the delay. I output the styles from the default template using `docfx.exe template export default`.  From the output, I renamed **docfx.css** to **default.css** and copied this and **docfx.js**, **docfx.vendor.css** and **docfx.vendor.js** to another folder, like:  

```
docFxTemplates\pdf\styles
…
    default.css
    docfx.js
    docfx.vendor.css
    docfx.vendor.js

```

Then in the **docfx.json** I updated the template section, like: 
```
"template": [
    "pdf.default",
    "docFxTemplates/pdf"
]
```

**Comment by ViceIce** [comment]
I had to add the fonts folder to my custom template too.

**Comment by ViceIce** [comment]
Additionally the markdown emojis are not working in pdf.

**Comment by superyyrrzz** [comment]
@MrBenMyster Thank you for the information. I agree it is ideal to work without customization. However, I am not sure whether it will break anything after this replacement. I need more investigation on this.

**Comment by superyyrrzz** [comment]
fixed in 2.56.3
